src/test/java/com/es/phoneshop/service/implementation/ProductServiceImplTest.java

```diff
@@ -73,10 +71,10 @@ public void getProduct_NullId_ShouldThrowIllegalArgumentException() {
     @Test
     public void findProducts_ShouldReturnProductListFromDao() {
         // when
-        List<Product> products = productService.findProducts();
+        productService.findProducts();
```

Comment:
Don't see defined behavior for productDao#findProducts method via when statement and checking that the test list with products specified in the thenReturn part is returned

Reply:
please, check the same for all tests